src/UI/admin/components/hall-request/AdminHallRequestStyle.js

```diff
@@ -11,4 +11,504 @@ const HallReqTitle = styled.h2`
   color: blue;
 `;
 
-export { HallReqContainer, HallReqTitle };
+const HallDiv = styled.div`
+  margin: auto;
+  width: 1024px;
+  height: 352px;
+  border-radius: 4px;
+  border: solid 1px #d9d9d9;
+  background-color: #ffffff;
+  display: flex;
+  box-sizing: border-box;
+`;
+
+const ImageDiv = styled.div`
+  height: 352px;
+  width: 268px;
+  display: flex;
+`;
+
+const TextDiv = styled.div`
+  height: 352px;
+  width: 504px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  box-sizing: border-box;
+`;
+
+const ButtonDiv = styled.div`
+  height: 352px;
+  width: 260px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  box-sizing: border-box;
+`;
+
+const ImageHall = styled.div`
+  width: 228px;
+  height: 136px;
+  border-radius: 2px;
+  border: solid 1px #229c7f;
+  margin-top: 16px;
+  margin-left: 16px;
+  object-fit: cover;
+`;
+const HeaderHall = styled.div`
+  width: 278px;
+  height: 32px;
+  font-family: Lora;
+  font-size: 20px;
+  font-weight: normal;
```

Comment:
Obrisati nepotrebne stilove. Sve što ima value normal vjerojatno ne treba explicitno definirati jer je to defaultna vrijednost...